api/src/test/java/marquez/jdbi/MarquezJdbiExternalPostgresExtension.java

```diff
@@ -34,6 +35,7 @@ public class MarquezJdbiExternalPostgresExtension extends JdbiExternalPostgresEx
     database = POSTGRES.getDatabaseName();
     plugins.add(new SqlObjectPlugin());
     plugins.add(new PostgresPlugin());
+    plugins.add(new Jackson2Plugin());
```

Comment:
Do we also need to add `Jackson2Plugin` to [`MarquezApp`](https://github.com/MarquezProject/marquez/blob/main/api/src/main/java/marquez/MarquezApp.java#L153)?

Reply:
Done.